src/openforms/scss/components/builder/_builder.scss

```diff
@@ -69,35 +69,33 @@ div.flatpickr-calendar.open {
   padding-top: 10px;
 }
 
-[ref='editForm'] {
```

Comment:
I think we had things scoped under this selector to prevent unexpected side effects with other unrelated components that happen to use the same classnames (it may have been django-admin-index). What's the reason for removing this?

Reply:
I removed because the the select dropdown in the editor has these classes but I couldn't find a parent with the `ref=editForm`. :thinking: